webapp/src/components/link_preview/preview.css

```diff
@@ -0,0 +1,156 @@
+:root {
+    --light-gray: #6a737d;
+    --blue: #274466;
+    --light-blue: #eff7ff;
+    --github-merged: #6f42c1;
+    --github-closed: #cb2431;
+    --github-open: #28a745;
+    --github-not-planned: #6e7681
+}
+
+@media (min-width: 544px) {
+    .github-preview--large {
+        min-width: 320px;
+    }
+}
+
+/* Github Preview */
+.github-preview {
```

Comment:
With sass we can nest everything under this class name in this file

Reply:
Oh I founded it can use [with CSS](https://caniuse.com/?search=nesting).  May I work with it?